build_runner/pubspec.yaml

```diff
@@ -12,7 +12,7 @@ dependencies:
   async: ">=1.13.3 <3.0.0"
   build: ">=1.0.0 <1.2.0"
   build_config: ">=0.4.0 <0.4.1"
-  build_daemon: ^0.6.0
+  build_daemon: ^0.7.0-dev
```

Comment:
You will want to add a dependency_override for this as well

Reply:
Done. (I think this is what you meant...)